torch_ort_inference/docker/README.md

```diff
@@ -0,0 +1,53 @@
+# OpenVINO™ integration with PyTorch ONNX Runtime Dockerfiles for Ubuntu* 18.04 and Ubuntu* 20.04
+
+
+We provide Dockerfiles for Ubuntu* 18.04 and Ubuntu* 20.04 which can be used to build runtime Docker* images for OpenVINO™ integration with PyTorch on CPU, iGPU and VPU.
+They contain all required runtime python packages, and shared libraries to support execution of a PyTorch Python app with the OpenVINO™ backend. By default, it hosts an Image Classification and Sequence Classification sample that demonstrate the performance benefits of using OpenVINO™ integration with PyTorch.
+
```

Comment:
Missing Pre-Requisites for Windows - WSL and relevant links.  Pre-Requisites for MacOS versions and Ubuntu versions.

Reply:
fixed